dom.bs

```diff
@@ -6306,16 +6303,12 @@ value of these steps:
 <p class=note>User agents could optimize <a for=Element>qualified name</a> and
 <a for=Element>HTML-uppercased qualified name</a> by storing them in internal slots.
 
-<p>To
-<dfn export id=concept-create-element lt="create an element|creating an element">create an element</dfn>,
-given a <var>document</var>, <var>localName</var>, <var>namespace</var>, and optional
-<var>prefix</var>, <var>is</var>, and <var>synchronous custom elements flag</var>, run these steps:
+<p>To <dfn export id=concept-create-element>create an element</dfn>, given a <a for=/>document</a>
```

Comment:
Why this instead of just `<a>document</a>`, as you did in https://github.com/whatwg/dom/pull/1307#discussion_r1881966012?

Reply:
`<a for=/>document</a>` is probably better for future proofing. Just in case something gets a document member.